clio-client/src/main/scala/org/broadinstitute/clio/client/commands/ClioCommand.scala

```diff
@@ -262,6 +263,7 @@ final case class DeliverArrays(
   force: Boolean = false
 ) extends DeliverCommand(ArraysIndex) {
   override val metadataMover = new ArrayDeliverer
+  override def deliverSampleLevelMetrics: Boolean = false
```

Comment:
sample level metrics are specific to crams so this option is not appropriate for arrays

Reply:
It will never appear as a command line option for arrays. Since its defined as a `def` and not in the parameter list, it will always be false and cannot be changed at runtime.